src/components/buttons/z-button/index.tsx

```diff
@@ -26,42 +32,45 @@ export class ZButton {
   @Prop({ reflect: true }) disabled?: boolean = false;
   /** HTML button type attribute. */
   @Prop() type?: HTMLButtonElement["type"] = ButtonTypeEnum.button;
-  /** Graphical variant: `primary`, `secondary`, `tertiary`, `dark-bg`. Defaults to `primary`. */
-  @Prop({ reflect: true }) variant?: ButtonVariantBean = ButtonVariantEnum.primary;
+  /** Graphical variant: `primary`, `secondary`, `tertiary`. Defaults to `primary`. */
+  @Prop({ reflect: true }) variant?: ButtonVariantBean =
+    ButtonVariantEnum.primary;
   /** `z-icon` name to use (optional). */
   @Prop() icon?: string;
   /** Available sizes: `big`, `small` and `x-small`. Defaults to `big`. */
   @Prop({ reflect: true })
   size?: ButtonSizeEnum = ButtonSizeEnum.big;
-  /** Reduce button size (deprecated).
-   * @deprecated Use `size` prop.
-   */
-  @Prop({ reflect: true }) issmall?: boolean = false;
-  /** Spy to render square button. */
+  /** set label text  */
   @Prop({ reflect: true })
-  square?: boolean = false;
+  text?: string;
 
   render() {
-    this.hostElement.style.pointerEvents = this.disabled ? "none" : "auto";
-
-    return (
-      <slot name="element">
-        <button
+    if (this.href)
+      return (
+        <a
+          aria-label={this.ariaLabel}
+          href={this.href}
+          target={this.target}
           id={this.htmlid}
-          name={this.name}
-          type={this.type}
-          disabled={this.disabled}
-          class={classNames(
-            this.variant,
-            this.size,
-            { issmall: this.issmall },
-            { square: this.square }
-          )}
+          class={classNames(this.variant, this.size)}
         >
           {this.icon && <z-icon name={this.icon} width={16} height={16} />}
-          <slot />
```

Comment:
Stavo ripensando alla rimozione totale dello slot...forse per coerenza con il tag button nativo di html converrebbe mantenere uno slot per la label del pulsante, in modo che sia possibile fare `<z-button>Click me</z-button>`